src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -47,11 +41,11 @@ public HttpApiHandler(DurableTaskExtension config, TraceWriter traceWriter)
 
 
         internal async Task<HttpResponseMessage> WaitForCompletionOrCreateCheckStatusResponseAsync(
-            HttpRequestMessage request,
-            string instanceId,
-            OrchestrationClientAttribute attribute,
-            TimeSpan timeout,
-            TimeSpan retryInterval)
+             HttpRequestMessage request,
```

Comment:
Extra whitespace padding was added to these parameters? It was already 4 spaces and now it is 5.

Reply:
Thank you for noticing that. I will fix it.